item-spec/item-spec.md

```diff
@@ -91,13 +91,13 @@ allowed to add additional fields such as a `title` and `type`.
 
 The following types are commonly used as `rel` types in the Link Object of an Item:
 
-| Type    | Description                                                  |
-| ------- | ------------------------------------------------------------ |
-| self    | **REQUIRED.** _Absolute_ URL to the item file itself. This is required, to represent the location that the file can be found online. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
-| root    | URL to the root [STAC Catalog](../catalog-spec/) or [Dataset](../dataset-spec/). |
-| parent  | URL to the parent [STAC Catalog](../catalog-spec/) or [Dataset](../dataset-spec/). |
-| dataset | **REQUIRED.** URL to a [STAC Dataset](../dataset-spec/). |
-| derived_from | URL to a STAC `Item` that was used as input data in the creation of this `Item` |
+| Type         | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| self         | **REQUIRED.** _Absolute_ URL to the item file itself. This is required, to represent the location that the file can be found online. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
+| root         | URL to the root STAC [Catalog](../catalog-spec/) or [Collection](../collection-spec/). |
+| parent       | URL to the parent STAC [Catalog](../catalog-spec/) or [Collection](../collection-spec/). |
+| collection   | STRONGLY RECOMMENDED. URL to a [Collection](../collection-spec/), which may use the [Commons extension](../extensions/commons/) and holds common fields of this and other Items (see chapter 'Collections'). |
```

Comment:
Should be required if Items are required to link to a collection, or is only the link to be required?

Reply:
See my other comments for an explanation.